styles/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    font-family: 'Roboto', sans-serif;
+    color: #383838;
+    margin: 0;
+}
+.offer-container {
+    max-width: 1440px;
+    height: 569px;
```

Comment:
**Just recommendation**
it's a bad practice to add `height` - try always to escape this if it's possible. Or use `min-height` instead of just `height`

Reply:
Corrected